controllers/userC.js

```diff
@@ -0,0 +1,26 @@
+const User = require('../models/user');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const asyncWrapper = require('../middleware/async');
+const {createCustomError} = require('../errors/custom_error');
```

Comment:
```suggestion
const { createCustomError } = require('../errors/custom_error');
```
Update Everywhere you are exporting in a a curly branch.
Always leave a space before you first alphabet and a space after the last alphabet